pom.xml

```diff
@@ -73,9 +73,9 @@
       </dependency>
 
       <dependency>
-        <groupId>io.camunda</groupId>
+        <groupId>io.camunda.spring</groupId>
         <artifactId>spring-zeebe-connector-runtime</artifactId>
-        <version>${version.spring-zeebe}</version>
+        <version>8.2.0-alpha2</version>
```

Comment:
Just out of curiosity: why was this change necessary? Why can't we resolve the `version.spring-zeebe` property?

Reply:
It's not necessary - my bad 🙂 Thanks for pointing out!